Mathlib/Topology/UniformSpace/Defs.lean

```diff
@@ -213,83 +212,70 @@ theorem subset_iterate_compRel {s t : SetRel α α} (h : idRel ⊆ s) (n : ℕ)
 def IsSymmetricRel (V : SetRel α α) : Prop :=
   Prod.swap ⁻¹' V = V
 
-set_option linter.deprecated false in
-@[deprecated (since := "2025-03-05")]
-alias SymmetricRel := IsSymmetricRel
-
 /-- The maximal symmetric relation contained in a given relation. -/
-@[deprecated SetRel.symmetrize (since := "2025-03-05")]
+@[deprecated SetRel.symmetrize (since := "2025-10-17")]
 def symmetrizeRel (V : SetRel α α) : SetRel α α :=
   V ∩ Prod.swap ⁻¹' V
 
 set_option linter.deprecated false in
-@[deprecated SetRel.isSymm_symmetrize (since := "2025-03-05")]
+@[deprecated SetRel.isSymm_symmetrize (since := "2025-10-17")]
 theorem symmetric_symmetrizeRel (V : SetRel α α) : IsSymmetricRel (symmetrizeRel V) := by
   simp [IsSymmetricRel, symmetrizeRel, preimage_inter, inter_comm, ← preimage_comp]
 
 set_option linter.deprecated false in
-@[deprecated SetRel.symmetrize_subset_self (since := "2025-03-05")]
+@[deprecated SetRel.symmetrize_subset_self (since := "2025-10-17")]
 theorem symmetrizeRel_subset_self (V : SetRel α α) : symmetrizeRel V ⊆ V :=
   sep_subset _ _
 
 set_option linter.deprecated false in
-@[deprecated SetRel.symmetrize_mono (since := "2025-03-05")]
-theorem symmetrize_mono {V W : SetRel α α} (h : V ⊆ W) : symmetrizeRel V ⊆ symmetrizeRel W :=
-  inter_subset_inter h <| preimage_mono h
-
-set_option linter.deprecated false in
-@[deprecated SetRel.comm (since := "2025-03-05")]
+@[deprecated SetRel.comm (since := "2025-10-17")]
 theorem IsSymmetricRel.mk_mem_comm {V : SetRel α α} (hV : IsSymmetricRel V) {x y : α} :
```

Comment:
This was originally meant to be removed, but it was used in `IsSymmetricRel.preimage_prodMap`, which was only deprecated in #23181, so I pushed the deprecation back.

**edit**: whoops, I accidentally changed the order here; will fix and repost this.